### ISSUE TITLE
Default image

### ISSUE THREAD

**Issue by tmtung144** [dietercoopman/smart#26]
**Is your feature request related to a problem? Please describe.**
I will always get 1 error if the photo doesn't exist

![image](https://user-images.githubusercontent.com/19670776/159298649-2495ad9a-abb7-4573-a7b0-b42196a02156.png)

**Describe the solution you'd like**
Please do a **try catch** or something to always return 1 default image (image can be white or have something like http://placehold.jp/150x150.png)

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by dietercoopman** [comment]
Hi, this problem is solved in version 1.6.4

Have a nice day

**Comment by dietercoopman** [comment]
default a 1x1 pixel is returned , you can change this behaviour via the config value 'file-not-found' , this is now the default config 

```php
<?php

return [
    'image'    => [
        'path'      => 'smart',
        'templates' => [
            'small' => [
                'resize' => [200, null, ['aspectRatio']],
            ],
            'big'   => [
                'resize' => [500, null, ['aspectRatio']],
            ]
        ],
        'file-not-found' => 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAAAXNSR0IArs4c6QAAAA1JREFUGFdj+P///38ACfsD/QVDRcoAAAAASUVORK5CYII='
    ],
    'download' => [
        'path'         => 'smart/downloads',
        'default-text' => 'download this file'
    ]
];
```
